src/commands/workspace/start.ts

```diff
@@ -42,12 +42,22 @@ export default class Start extends Command {
       description: 'workspace name: overrides the workspace name to use instead of the one defined in the devfile. Works only for devfile',
       required: false,
     }),
+    'access-token': string({
+      description: 'Che OIDC Access Token',
+      env: 'CHE_ACCESS_TOKEN'
+    }),
     'listr-renderer': string({
       description: 'Listr renderer. Can be \'default\', \'silent\' or \'verbose\'',
       default: 'default'
     }),
   }
 
+  async checkToken(flags: any, ctx: any) {
+    if (ctx.isAuthEnabled && !flags['access-token']) {
+      this.error('E_AUTH_REQUIRED - Che authentication is enabled and an access token need to be provided (flag --access-token).')
```

Comment:
Maybe tell the user in few words, how to get that token ?

Reply:
there is a link in server:stop command but it's about che6 doc
https://www.eclipse.org/che/docs/che-6/authentication.html

I would say that we should have like openshift a 'copy login' command or something like that from dashboard UI

Reply:
Could you create an issue for that (adding a 'copy login' from dashboard UI) ?

Reply:
https://github.com/eclipse/che/issues/13852